pkg/utils/timeutils.go

```diff
@@ -44,6 +44,8 @@ func Str2duration(duration string) (int64, error) {
 				multiply = 24 * 3600 * 1000
 			case "w":
 				multiply = 7 * 24 * 3600 * 1000
+			case "M":
+				multiply = 30 * 24 * 3600 * 1000
```

Comment:
What about months/years with different number of days? Should be resolved by clendar